Algorithms/Search/AStar/VecN.cs

```diff
@@ -7,6 +8,8 @@ namespace Algorithms.Search.AStar
     /// </summary>
     public struct VecN : IEquatable<VecN>
     {
+        public const double DefinedAccuracy = 0.000001;
```

Comment:
It's better to move constants into const variables or configs rather than to use them right in the methods.